source/crud/query/specify-documents-to-return.txt

```diff
@@ -25,4 +25,300 @@ operation by using the following methods:
+
+Sometimes, the order of two or more documents is ambiguous using a
+specified sort. In the above case, both "A Dance with Dragons" and
+"Infinite Jest" have ``1104`` pages, so the order in which they are
+returned is not guaranteed. To resolve ties in your sorted results in a
+repeatable way, add more fields to the sort document:
+
+.. code-block:: javascript
+   :emphasize-lines: 4
+
+   // define an empty query document
+   const query = {};
+   // sort in ascending (1) order by length
+   const sort = { length: 1, author: 1 };
+   const cursor = myColl.find(query).sort(sort);
+   for await (const doc of cursor) {
+     console.dir(doc);
+   }
+
+With the addition of the ``author`` field to the sort document, the read operation sorts
+matching documents first by ``length`` then, if there is a tie, by ``author``. Matched
+document fields are compared in the same order as fields are specified in the sort
+document. ``find()`` returns the following ordering of documents when this sort is used on
+the documents matching the query, sorting ``"Martin"`` before ``"Wallace"`` for the two books with
+the same length:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "The Brothers Karamazov", "author": "Dostoyevsky", "length": 824 }
+   { "_id": 5, "title": "Cryptonomicon", "author": "Stephenson", "length": 918 }
+   { "_id": 3, "title": "Atlas Shrugged", "author": "Rand", "length": 1088 }
+   { "_id": 6, "title": "A Dance with Dragons", "author": "Martin", "length": 1104 }
+   { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
+   { "_id": 2, "title": "Les Misérables", "author": "Hugo", "length": 1462 }
+
+Combine Limit, Sort, and Skip
+-----------------------------
+
+You can combine ``limit``, ``sort``, and ``skip`` in a single
+operation. This allows you to set a maximum number of sorted documents to
+return, skipping a specified number of documents before returning.
+
+The following example returns documents with the ``length`` value of
+``"1104"``. The results are sorted in alphabetical order, skipping the first
+document:
+
+.. code-block:: javascript
+   :emphasize-lines: 5
+
+   // define a query to look for length value of 1104
+   const query = {length: "1104"};
+   const options = {
+      // sort in alphabetical (1) order by title
+      sort : { title: 1 },
+      // omit the first document
+      skip : 1,
+   }
+   const cursor = myColl.find(query).sort(sort);
+   for await (const doc of cursor) {
+     console.dir(doc);
+   }
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
+
+.. note::
+
+   The order in which you call these methods doesn't change the documents
+   that are returned. The driver automatically reorders the calls to perform the
+   sort and skip operations first, and the limit operation afterward.
+
+You can also limit, sort, and skip results by specifying them as
```

Comment:
I: the previous example does specify these as parameters (an options parameter) to the find() method. Either you can show how to specify each as methods chained to find(), or omit this part